e2e/README.md

```diff
@@ -0,0 +1,366 @@
+# Kubeflow End to End
+
+This example guides you through the process of taking a distributed model, modifying it to work with the tf-operator, providing data to your model, and serving the resulting trained model. We will be using Argo to manage the workflow, Kube Volume Controller to supply data via S3, Tensorflow's S3 support for saving model training info, Tensorboard to visualize the training, and Kubeflow to serve the model.
```

Comment:
KVC downloads data from S3 to local disks on node. This reduces the overall job time since the download will be re-used for subsequent runs (the mnist CR will not be created if it exists, so workloads can just keep using the existing object).

Also reads from a local disk will be faster than going over a network for most use cases (provided your data, or piece of data, can fit).

I can expand on the explanation, and also add a flag to skip KVC if someone wants to stream directly from S3.

Reply:
Why would there be subsequent runs? Aren't you just training once?

If the purpose of this example is to provide an E2E example to help onboard users to Kubeflow, then why include the KVC which is an experimental component?

Removing the KVC would simplify the example and make it less likely the example will break over time